tests/unit/ui/components/TestOverlayCard.js

```diff
@@ -4,13 +4,33 @@ import {
   renderIntoDocument,
 } from 'react-dom/test-utils';
 import { findDOMNode } from 'react-dom';
+import { Provider } from 'react-redux';
 
+import createStore from 'amo/store';
 import Overlay from 'ui/components/Overlay';
-import OverlayCard from 'ui/components/OverlayCard';
+import OverlayCard, { OverlayCardBase } from 'ui/components/OverlayCard';
 
 describe(__filename, () => {
+  const { store } = createStore();
+
+  const getProps = ({ ...props } = {}) => {
+    return {
+      className: 'OverlayCard',
+      id: 'OverlayCard',
+      store,
+      ...props,
+    };
+  };
+
   function render(props = {}) {
-    return renderIntoDocument(<OverlayCard {...props} />);
+    return findRenderedComponentWithType(
+      renderIntoDocument(
```

Comment:
I left this in place for now as initially I was focused on just the Overlay. 

also a note here, on why I removed some of this tests here:

When I provide this Provider wrapper, the `overlayContainer` isn't available anymore. You will see it removed in this PR but even with this added back, this is coming up undefined.

I was running into this similar issue too with hide, show, toggle - but I didn't even see these used here (or in Overlay - altho I left in Overlay for now).

the Overlay tests seem to do some of this already (such as check for default set up).

let me know if this is an issue